packages/workflows-sdk/src/helper/workflow-export.ts

```diff
@@ -211,7 +211,13 @@ function createContextualWorkflowRunner<
 
     let result
     if (options?.wrappedInput) {
-      result = await resolveValue(resultFrom, transaction.getContext())
+      result = resolveValue(resultFrom, transaction.getContext())
+      if (result instanceof Promise) {
+        result = await result.catch((e) => {
```

Comment:
I believe there is no need to await here (but of course is fine to do so)

Reply:
Yeah I ll keep it in case there is asynchronous function running